mautic/mautic
# [UX] Change "Published" to "Available for use" for bundles

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR addresses the labeling of some resources during their creation. The term “Published” has been updated to “Available for use” to more accurately reflect the resource’s status. This change ensures that users have a clear and precise understanding of when they become usable within the system. The new label eliminates any ambiguity, thereby improving the overall user experience, especially for those who are setting up resources for the first time or managing complex configurations.

Based on https://community.mautic.org/assemblies/ux-ui-tiger-team/f/59/debates/4

Bundles addressed here:

- [x] Custom fields
- [x] Category
- [x] Dynamic content
- [x] Emails
- [x] Marketing messages
- [x] Text messages
- [x] Landing pages
- [x] Assets
- [x] Points (Groups)
- [x] Reports



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open each item on the list above
3. Create a new
4. Look at the label

![image](https://github.com/mautic/mautic/assets/116097999/4eb395f3-9a42-4cde-a5f0-71461a2c7678)
